compiler/test/stdlib/rational.test.gr

```diff
@@ -0,0 +1,63 @@
+module RationalTest
+
+let (===) = (==)
+let (!==) = (!=)
+
+include "rational"
+from Rational use *
+
+assert fromNumber(1/2) == 1/2r
```

Comment:
Why no float tests?

Reply:
It doesn't look like this functionality is currently supported, with only ints being coercible to rationals https://github.com/grain-lang/grain/blob/3907668ce1452cf83aca964cbc1b496472a38e46/stdlib/runtime/numbers.gr#L2437

I'm also not exactly sure how you would implement this; the majority of floats would result in some very nasty rational numbers